recipes/getopt-for-visual-studio/all/conanfile.py

```diff
@@ -0,0 +1,42 @@
+from conans import ConanFile, tools
+from conans.errors import  ConanInvalidConfiguration
+import os
+
+
+class GetoptForVisualStudio(ConanFile):
+    name = "getopt-for-visual-studio"
+    description = "GNU getopt for Visual Studio"
+    topics = ("conan", "getopt", "cli", "command line", "options")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/skandhurkat/Getopt-for-Visual-Studi"
+    license = "MIT", "BSD-2-Clause"
+    exports_sources = "patches/**"
+    settings = "compiler"
```

Comment:
```suggestion
    settings = "compiler"
    no_copy_source = True
```

I think you can move the build step to package (patch)

Reply:
I added some patches. These won't work together with `no_copy_source=True`